### DISCUSSION TITLE
Welcome to forked goodbye-captcha discussions!

### DISCUSSION THREAD

**Discussion by knutsp** [Webfacing/goodbye-captcha#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you want to do in this community.

**Comment by knutsp** [comment]
This may be just temporary, as the original plugin author Mihai Chelaru (@[mihche](https://profiles.wordpress.org/mihche/)) might start maintaining it again.
I have kept the same slug and versioned this such that if original plugin author  releases a new version, it will be newer than this and replace this one on your site.
Version string: 3.1.43.{n}-wf where {n} is the sequential release number of this fork. "wf" is prefix of WebFacing, my GitHub organization (and company "Nettvendt", Norway).

**Reply by rseanmcgown** [reply]
I have actually tried reaching out to Mihai but never got a response. Have you heard if there are any plans to renew development? This is a go-to for me when I do sites.

**Comment by jotazzu** [comment]
To make the payed "WPBruiser - Gravity Forms" extension plugin v3.1.39 compatible with PHP 7.4 one line in file wp-bruiser-gravityforms/engine/modules/gravity-forms/GdbcGravityFormsPublicModule.php has to be changed:

GdbcGravityFormsPublicModule.php, line 61
Original:
`$gravityFormsVersion =  isset(GFForms::$version) ? GFForms::$version : isset(GFCommon::$version) ? GFCommon::$version : 0;`

Replace with (set parentheses for nested ternary operator for compatibility with PHP 7.4):
`$gravityFormsVersion =  isset(GFForms::$version) ? GFForms::$version : (isset(GFCommon::$version) ? GFCommon::$version : 0);`
